resources/guides/uwinsite_guide.md

```diff
@@ -0,0 +1,236 @@
+slug: /guides/uwinsite_guide
+---
+
+This is a guide on how to use and access the important sections on your UWinsite Student page.
+
+## Course Management
+
+### How to Plan & Enroll in Classes
+
+To plan your courses and register in classes in UWinsite Student, you will use the
+**Manage Classes** section on the Student Homepage.
+
+#### Planning your Classes
+
+You can start planning your classes and adding them to your Shopping Cart in UWinsite
+Student when the validation period opens. The validation period is when the timetable
+for the next semester is close to final and the courses being offered are visible in
+UWinsite Student. Typically, the validation period starts two to three weeks before
+registration begins.
+
+The preferred method for placing your classes into the Shopping Cart is through Class
+Search and Enroll. This method shows you just the courses that are offered during the
+term for which you are planning.
+
+**To get started planning your classes:**
+
+![img](../../static/img/uwinsite_guide/uwinsite_homepage.jpg)
+
+1. On the Student Homepage, click the Manage Classes tile.
+2. Click the Class Search & Enroll tab in the left navigation menu.
+![img](../../static/img/uwinsite_guide/class_search_and_enroll.jpg)
+3. Select the term in which you will be registering.
+4. In the Search for Classes field, enter a subject area, topic or course code.
+5. From the Search Results, select a course.
+6. On the **Course Information** page you will see all class sections for the term you
+selected. Some classes have one section while others have multiple sections. Under
+**Class Selection**, you can view the time, date, instructor, location and available seats
+for each section of the class.
+![img](../../static/img/uwinsite_guide/course_info_and_class_selection.jpg)
+7. Select a class section. You can click anywhere on the class except for the Lecture or
+Lab link to select it.
+8. You’ll now be directed to the confirmation process:
+    1. Review **Class Selection** and click **Next**
+    2. Review **Class Preferences**
+        + This is where you have the option to add yourself to the waitlist if the class is
+        full. Select **Yes** if this will not put you in course overload (*normally 5 classes, but
+        check your program*). 
+        + Select **No** if having this course on the waitlist would put you in course overload.
+        You will instead need to use **Swap Classes** so UWinsite Student knows what
+        course to drop if you are to reach the top of the waitlist.
+    3. Click **Accept**
+    4. On the **Enroll** or **Add to Cart** page, click the **Add to Shopping Cart** button and
+    then click **Next**.
+    5. On the **Review and Submit** page, review your selections and then click **Submit**.
+
+#### Registering in your Classes
+
+Registration at UWindsor is by appointment time. Registration appointment times are
+assigned based on program and level of study. Your registration appointment time is
```

Comment:
```suggestion
Registration at UWindsor is by appointment time. Registration appointment times are
assigned based a number of factors including program and level of study. Your registration appointment time is
```

I think this is better because its not based solely program and year.